uktrade/data-hub-frontend
# Reactify Large Capital Profile requirements section

## Description of change

This is the third stage of the work started in https://github.com/uktrade/data-hub-frontend/pull/5480

The requirements section of the Large Capital Profile page has now been reactified. I have focused on maintaining consistency with the other two React forms, which means that there is a small amount of duplication which will be resolved in the last PR.

## Test instructions

Open the large capital profile tab for a company and open the requirements section. You should be able to view and edit the requirements data.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
